WordPressKit/RemotePostAutosave.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+/// Encapsulates the autosave attributes of a post.
+@objc
+@objcMembers
+open class RemotePostAutosave: NSObject {
+    open var title: String?
+    open var excerpt: String?
+    open var content: String?
+    open var modifiedDate: Date?
+    open var identifier: NSNumber?
+    open var authorID: String?
+    open var postID: NSNumber?
+    open var previewURL: String?
```

Comment:
As I understand it, `open` would allow consumers of this library to create a subclass of `RemotePostAutosave`. I don't think we really need that. The `open` attribute would probably add some confusion about the intent. What do you think about making this class and its properties `public` instead? 

I experimented with `public internal(set)` but that does not seem to work well with Objective-C. 😞

Reply:
I missed that, updated now. 

I tested `internal(set)` now too. It didn't work either and I found a [SO question](https://stackoverflow.com/q/32700078/1305067) which linked to [this Apple doc](https://developer.apple.com/documentation/swift/imported_c_and_objective-c_apis/importing_swift_into_objective-c):

> Methods and properties that are marked with the internal modifier and declared within a class that inherits from an Objective-C class are accessible to the Objective-C runtime. However, they're inaccessible at compile time and don't appear in the generated header for a framework target.

So they're there at runtime - just not at compile time. 😔